src/librustc_data_structures/bitvec.rs

```diff
@@ -63,6 +62,45 @@ impl BitVector {
         }
     }
 
+    /// Return and unset first bit set.
+    pub fn pop(&mut self) -> Option<usize> {
```

Comment:
There’s a test for this function introduced by this PR (but not necessarily in the commit you commented on).